src/Utils/WordPress/appengine-wordpress

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+if (file_exists(__DIR__ . '/../../../vendor/autoload.php')) {
+    // Run the command from the git clone.
```

Comment:
Not sure I follow what you're getting at with these variations. The difference between a git clone and a composer install? Why not make it a composer global install?

Reply:
Just wanting to handle both the dev case (`git clone`) and the vendor case (`composer install`).

As for `global` install, I haven't decided if that's how I want to surface these tools. That won't affect these lines, though.